package.json

```diff
@@ -24,13 +24,14 @@
   "devDependencies": {
     "eslint": "^7.9.0",
     "eslint-config-origami-component": "^2.1.0",
-    "origami-ci-tools": "^2.1.0",
+    "origami-ci-tools": "3.0.0-beta.4",
+    "remark": "^12.0.1",
+    "remark-lint": "^7.0.1",
+    "remark-preset-lint-origami-component": "^2.0.0-beta.1",
+    "stylelint": "^13.7.0",
     "stylelint-config-origami-component": "^1.0.4",
     "typescript": "^4.0.5"
   },
```

Comment:
I think we need to define an engines property which specifies this package depends on npm version 7 or higher like this:
https://github.com/Financial-Times/o-test-component/blob/27f60686d31f0a161e29ccd9d1f969e1b77e68d1/package.json#L13-L15